common/build.gradle

```diff
@@ -11,7 +11,7 @@ apply plugin: 'maven-publish'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "26.0.2"
+    buildToolsVersion '27.0.3'
```

Comment:
The `.travis.yml` needs to be updated to support this -- good to see this getting rev'd though 😁